controllers/whoToFollow/whoToFollow.js

```diff
@@ -34,9 +34,11 @@ module.exports = async (req, res) => {
   }
 
   try {
-    const topicsQuery = `SELECT * FROM vwm_user_topic_follower_count_rank WHERE topic_id IN (:topics) AND topic_follower_rank <= 10 order BY topic_follower_rank ASC`;
+    const topicsQuery = `SELECT * FROM vwm_user_topic_follower_count_rank WHERE topic_id IN (:topics) AND topic_follower_rank <= 10 ORDER BY topic_follower_rank ASC, CASE WHEN profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END`;
 
-    const locationQuery = `SELECT * FROM vwm_user_location_follower_count WHERE location_id IN (:locations) AND follower_rank <= 10 order BY follower_rank ASC`;
+    const locationQuery = `SELECT * FROM vwm_user_location_follower_count WHERE location_id IN (:locations) AND follower_rank <= 10 ORDER BY follower_rank ASC, CASE WHEN profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END`;
+
+    const otherTopicsQuery = `SELECT * FROM vwm_user_topic_follower_count_rank WHERE topic_id NOT IN (:topics) AND topic_follower_rank <= 10 ORDER BY follower_count DESC, CASE WHEN profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END LIMIT 35`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL queries are directly using string interpolation to insert parameters into the query. This can lead to SQL injection attacks if the input is not properly sanitized. It's better to use parameterized queries or prepared statements to prevent this security issue.
```diff
-    const topicsQuery = `SELECT * FROM vwm_user_topic_follower_count_rank WHERE topic_id IN (:topics) AND topic_follower_rank <= 10 ORDER BY topic_follower_rank ASC, CASE WHEN profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END`;
+    const topicsQuery = `SELECT * FROM vwm_user_topic_follower_count_rank WHERE topic_id IN (?) AND topic_follower_rank <= 10 ORDER BY topic_follower_rank ASC, CASE WHEN profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->